Koha/Plugin/EDS/js/EDSScript.js

```diff
@@ -200,6 +200,7 @@ $.each($.cookie('bib_list').split("/"), function () {
 });
 
 function StartEDS(edsLang) {
+	//console.log("Running startEDS...");
```

Comment:
do you want to remove the log here?

Reply:
I may need it for some future testing, but once I've removed the remaining defects I'll do a big cleanup of any remaining console logs.  Thanks for pointing it out!